test/active_job/queue_adapters/resque_test.rb

```diff
@@ -3,7 +3,24 @@
 class ActiveJob::QueueAdapters::ResqueTest < ActiveSupport::TestCase
   include ActiveJob::QueueAdapters::AdapterTesting
 
+  test "it supports queue pausing when ResquePauseHelper is defined" do
```

Comment:
First time using minitest since I've used mostly rspec. I noticed the test descriptions in other files do not quite follow the same "it ..." conventions as rspec, so these could be renamed if they don't make sense.

Also, feel free to let me know if we should move these to another file, placing them inside `ActiveJob::QueueAdapters::AdapterTesting::QueryJobs` didn't make sense to me since those are reusable while these tests are adapter-specific but maybe it makes sense to move them into a separate file.

Reply:
No worries! I think the tests are ok here, agree on not adding this into any of the specific `AdapterTesting::*` modules. 

About the conventions for the tests, that's right, with minitest usually `it` is omitted. I'd rename these tests just removing the `it`, so they'd be: 
```suggestion
  test "supports queue pausing when ResquePauseHelper is defined" do
```

Reply:
sg! just pushed the latest changes